independent-projects/tools/registry-client/src/main/java/io/quarkus/maven/StreamCoords.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.maven;
+
+/**
+ * @deprecated in favor of {@link io.quarkus.registry.catalog.PlatformStreamCoords}
+ */
+@Deprecated
+public class StreamCoords extends io.quarkus.registry.catalog.PlatformStreamCoords {
```

Comment:
I think this can be removed completely? It's not being used anywhere AFAIK

Reply:
You are most probably right. I kept it, just in case, for backward compatibility.

Reply:
Right, my point is that deleting it shouldn´t break backward compatibility since it's already well encapsulated from the public APIs. It's unlikely that someone is using it since it was introduced less than 2 months ago 😉

Reply:
It shouldn't affect our dev tools, so yes, I think it's safe to remove it.

Reply:
Removed it.

Reply:
@ia3andy just to confirm, this class is not used in code.quarkus.io, right?

Reply:
It doesn't look like it.